src/components/Button/GlobalButton/Button.module.css

```diff
@@ -1,12 +1,12 @@
 .btn {
-  padding: 8px 15px;
+  padding: 8px 25px;
   text-align: center;
   cursor: pointer;
-  border-radius: 8px;
+  border-radius: 40px;
```

Comment:
The radius might be different at different places so make another variant that also defines the border radius of the button for example you can make a class called `rounded-button` which has a different border radius and the default button can have the radius that we previously had.

We can use this class in places where we need it.

Reply:
U mean hide contribute to milan button?

Reply:
So basically I create a another button with different border-radius...